src/interfaces/ISUSDS.sol

```diff
@@ -1,8 +1,8 @@
 // SPDX-License-Identifier: AGPL-3.0-or-later
 pragma solidity >=0.8.0;
 
-interface IPot {
+interface ISUSDS {
     function ssr() external view returns (uint256);
-    function chi() external view returns (uint256);
-    function rho() external view returns (uint256);
+    function chi() external view returns (uint192);
+    function rho() external view returns (uint64);
```

Comment:
Why these uint sizes and uint sizes in `ISSRORacle` (`SUSDSData` struct) differ?

Reply:
Well `ISUSDS` is just the format that PullUp chose. We pack them into 1 word to make it 1 SLOAD for the oracle.